lib/wings/active_fedora_converter.rb

```diff
@@ -84,6 +84,12 @@ def id
       resource.alternate_ids.first.to_s
     end
 
+    def self.DefaultWork(resource_class)
```

Comment:
This method looks a little suspicious. And I'm trying to figure out why you wouldn't want a private instance method instead of using a class method.

Reply:
Now I'm seeing.